importer/handlers/shapefile/handler.py

```diff
@@ -151,15 +152,31 @@ def create_ogr2ogr_command(files, original_name, ovverwrite_layer, alternate):
         )
         layers = ogr.Open(files.get("base_file"))
         layer = layers.GetLayer(original_name)
-        additional_option = (
+
+        if not files.get("cpg_file"):
+            # set a default
+            encoding = "ISO-8859-1"
```

Comment:
@ridoo in case no cpg or cst files shouldn't we leave the encoding out and just use the system encoding?